lib/generator._js

```diff
@@ -95,6 +96,18 @@ function ScriptGenerator(repositoryRoot, projectType, projectPath, solutionPath,
     var relativeSolutionPath = path.relative(repositoryRoot, solutionPath);
     log.info('Solution file path: .' + path.sep + relativeSolutionPath);
     this.solutionPath = relativeSolutionPath;
+
+    // find .xproj file, should be under projectPath
```

Comment:
move this logic into `generateAspNetCoreDeploymentScript ` since it's specific to AspNetCore